pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -171,6 +180,32 @@ func (c *Cluster) Start() error {
 		logger.Infof("mgr metrics service started")
 	}
 
+	// enable monitoring if `monitoring: enabled: true`
+	if c.monitoringSpec.Enabled {
+		if c.clusterInfo.CephVersion.IsAtLeastNautilus() {
+			logger.Debugf("starting monitoring deployment")
+			// servicemonitor takes some metadata from the service for easy mapping
+			if err := c.enableServiceMonitor(service); err != nil {
+				logger.Errorf("failed to enable service monitor. %+v", err)
+			} else {
+				logger.Infof("servicemonitor enabled")
+			}
+			// namespace in which the prometheusRule should be deployed
+			// if left empty, it will be deployed in current namespace
+			namespace := c.monitoringSpec.RulesNamespace
+			if namespace == "" {
+				namespace = c.Namespace
+			}
+			if err := c.deployPrometheusRule(prometheusRuleName, namespace); err != nil {
+				logger.Errorf("failed to deploy prometheusRule. %+v", err)
+			} else {
+				logger.Infof("prometheusRule deployed")
+			}
+			logger.Debugf("ended monitoring deployment")
+		} else {
+			logger.Debugf("monitoring not supported for <v%v", c.clusterInfo.CephVersion.Major)
```

Comment:
the message could be clearer

Reply:
any suggestions?

Reply:
`monitoring not supported for ceph version prior to v%v`?

Reply:
done.